### ISSUE TITLE
preparation for node v0.8

### ISSUE THREAD

**Issue by adrai** [schakko/node-usb#12]
Have you started to use node-gyp (https://github.com/TooTallNate/node-gyp)? This will be needed for the next stable node version.

**Comment by schakko** [comment]
Nope, have not started yet. Currently I am too busy to do something for node-usb :/
Trying to take a look in July.

**Comment by adrai** [comment]
ok, thanks... ;-)
